src/main/java/org/opensearch/knn/index/memory/NativeMemoryLoadStrategy.java

```diff
@@ -167,11 +167,15 @@ public NativeMemoryAllocation.TrainingDataAllocation load(
             NativeMemoryAllocation.TrainingDataAllocation trainingDataAllocation = new NativeMemoryAllocation.TrainingDataAllocation(
                 executor,
                 0,
-                nativeMemoryEntryContext.calculateSizeInKB()
+                nativeMemoryEntryContext.calculateSizeInKB(),
```

Comment:
Shouldn't we modify this method? the size might be different for byte vector?

Reply:
ack